lib/signaling/v2/room.js

```diff
@@ -142,6 +145,13 @@ class RoomV2 extends RoomSignaling {
     this._initNetworkQualityMonitorSignaling();
     this._initPublisherHintSignaling();
 
+    this._otherMediaSignalings.forEach(({ MediaSignaling, name, initMethod }) => {
```

Comment:
same comment here. I prefer not making these runtime properties, and instead keep them in separate map.